src/popup/pages/profile.tsx

```diff
@@ -178,7 +179,10 @@ export const Profile = ({ username }: { username: string }) => {
                         <div className="flex flex-col items-center justify-center p-2 text-xs">
                             <p>Contributed Repos</p>
 
-                            <p className="font-medium text-5xl">-</p>
+                            <p className="font-medium text-5xl">
+                                {/* eslint-disable-next-line @typescript-eslint/no-unsafe-call*/}
```

Comment:
Can you check if you can fix this without disabling eslint?

Reply:
ok, i have been trying this for past one hour, and I have still no idea, how to do this.
the error I keep getting is 
Unsafe call of an `any` typed value.

but I have already ensured that my function or userPR is not 'any' typed value. you can see that in the latest commit.
still the error isn't going away.